electron-builder.json

```diff
@@ -41,11 +41,11 @@
   },
   "directories": {
     "buildResources": "build",
-    "output": "package_electron"
+    "output": "build_electron"
```

Comment:
please do not remove the build_electron folder. keep vite as it is for now, we've made some augmentations that could makes the bundles much much smaller and faster.

Reply:
@xyloflake I suggest you making these changes in separate PR.